### DISCUSSION TITLE
Command processing

### DISCUSSION THREAD

**Discussion by Kachankapusta** [rubenlagus/TelegramBots#1110] [Q&A]
Is there a way to process the user click on the "delete chat" button by the bot itself?

![image](https://user-images.githubusercontent.com/38657459/184864294-7c191321-6f48-41a3-aaf6-a3aa36b12c6a.png)

I would like to process the user's departure from the bot and make an appropriate entry in the database so that when he reconnects to the bot, he will be sent a menu different from the users who joined for the first time. What special link should be processed by the bot and what method to process it? If this functionality is possible at all.

**Comment by MRiabov** [comment]
Well, I just know that the user can _stop_ a bot. Also, why not just check on /start if the user is registered or not? It's simple, just look up the database...
